keyboards/skeletonkbd/frost68/info.json

```diff
@@ -0,0 +1,124 @@
+{
+    "manufacturer": "skeletonkbd",
+    "keyboard_name": "Frost68",
+    "maintainer": "skeletonkbd",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+	"matrix_pins": {
+		"cols": [
+			"GP21",
+			"GP22",
+			"GP23",
+			"GP24",
+			"GP25",
+			"GP26",
+			"GP28",
+			"GP3",
+			"GP4",
+			"GP5",
+			"GP6",
+			"GP9",
+			"GP10",
+			"GP11",
+			"GP12",
+			"GP13"
+
+		],
+		"rows": [
+			"GP29",
+			"GP27",
+			"GP20",
+			"GP19",
+			"GP18"
+		]
+	},
+    "processor": "RP2040",
+    "url": "",
```

Comment:
Missing the URL

Reply:
add url in info.json
1da3f3af8cbba958a8889868f43bbdf218fc56ca